src/main/java/apoc/export/csv/CsvFormat.java

```diff
@@ -96,10 +98,8 @@ public ProgressInfo dump(Result result, Writer writer, Reporter reporter, Export
             result.accept((row) -> {
                 for (int col = 0; col < header.length; col++) {
                     Object value = row.get(header[col]);
-                    data[col]= FormatUtils.toString(value);
```

Comment:
do we really need manual quoting here? I thought CsvWriter does that for us?

Reply:
you're right. It should now be better.